tests/microos/networkmanager.pm

```diff
@@ -99,10 +108,18 @@ true'
     ($RcManager, $mode) = dns_mgr();
     die 'wrong DNS-Manager is currently used for systemd-resolved' if ($RcManager !~ /unmanaged/ || $mode !~ /systemd-resolved/);
     ping_check;
+    # Restore the original system config
+    restore_config;
 }
 
 sub test_flags {
-    return {always_rollback => 1};
+    return {fatal => 1};
+}
+
+sub post_fail_hook {
+    restore_config;
+    script_run("journalctl --no-pager -o short-precise > /tmp/full_journal.log");
+    upload_logs "/tmp/full_journal.log";
```

Comment:
Let's use a browser-friendly extension so we don't have to download to open:

```suggestion
    script_run("journalctl --no-pager -o short-precise > /tmp/full_journal.txt");
    upload_logs "/tmp/full_journal.txt";
```

Reply:
Thanks